notebooks/py_scripts/2022-allen-institute-workshop.py

```diff
@@ -0,0 +1,201 @@
+                  'h5py': [],
+                  'h5py_key': 'data',
+                  'save_path0': [],
+                  'subfolders': [],
+                  'nplanes': 1,
+                  'nchannels': 1,
+                  'functional_chan': 1,
+                  'tau': 1.0,
+                  'fs': 10.0,
+                  'force_sktiff': False,
+                  'preclassify': 0.0,
+                  'save_mat': False,
+                  'combined': True,
+                  'aspect': 1.0,
+                  'do_bidiphase': False,
+                  'bidiphase': 0.0,
+                  'do_registration': True,
+                  'keep_movie_raw': False,
+                  'nimg_init': 300,
+                  'batch_size': 500,
+                  'maxregshift': 0.1,
+                  'align_by_chan': 1,
+                  'reg_tif': False,
+                  'reg_tif_chan2': False,
+                  'subpixel': 10,
+                  'smooth_sigma': 1.15,
+                  'th_badframes': 1.0,
+                  'pad_fft': False,
+                  'nonrigid': True,
+                  'block_size': [128, 128],
+                  'snr_thresh': 1.2,
+                  'maxregshiftNR': 5.0,
+                  '1Preg': False,
+                  'spatial_hp': 50.0,
+                  'pre_smooth': 2.0,
+                  'spatial_taper': 50.0,
+                  'roidetect': True,
+                  'sparse_mode': False,
+                  'diameter': 10,
+                  'spatial_scale': 0,
+                  'connected': True,
+                  'nbinned': 5000,
+                  'max_iterations': 20,
+                  'threshold_scaling': 1.0,
+                  'max_overlap': 0.75,
+                  'high_pass': 100.0,
+                  'inner_neuropil_radius': 2,
+                  'min_neuropil_pixels': 350,
+                  'allow_overlap': False,
+                  'chan2_thres': 0.65,
+                  'baseline': 'maximin',
+                  'win_baseline': 60.0,
+                  'sig_baseline': 10.0,
+                  'prctile_baseline': 8.0,
+                  'neucoeff': 0.7,
+                  'xrange': np.array([0, 0]),
+                  'yrange': np.array([0, 0])}
+
+imaging.ProcessingParamSet.insert_new_params(processing_method='suite2p', 
+                                             paramset_idx=1, 
+                                             params=params_suite2p,
+                                             paramset_desc='diameter=10')
+
+imaging.ProcessingParamSet()
+
+# +
+os.makedirs('subject3/210107_run00_orientation_8dir/suite2p_1', exist_ok=True)
+
+imaging.ProcessingTask.insert1(dict(subject='subject3', 
+                                    session_datetime='2022-09-01 19:16:44', 
+                                    scan_id=0,
+                                    paramset_idx=1,
+                                    processing_output_dir='subject3/210107_run00_orientation_8dir/suite2p_1'))
```

Comment:
See above re absolute path on code book `/home/inbox/{djarchive-version}`

Reply:
I believe that the directory attribute within the table should be treated as a relative path, but I could be mistaken.

Reply:
oh, no, you're right. relative in this table